e2e/utils/index.ts

```diff
@@ -399,7 +398,32 @@ export async function cleanupProject(opts?: RunCmdOpts) {
 }
 
 export function runCypressTests() {
-  return process.env.NX_E2E_RUN_CYPRESS === 'true';
+  if (process.env.NX_E2E_RUN_CYPRESS === 'true') {
```

Comment:
I see this is only called in
angular-core
next
react
storybook-angular

but this will still have an issue with e2e cypress tests at least (idk if there are other). so we'll need to also call it there in those other tests that use cypress.

Reply:
Let's check if this solves those and then we can add this wrapper function to other missing places in a followup PR.